README.md

```diff
@@ -8,6 +8,15 @@ A youtube downloader written in Javascript. To be used with the command line. If
 
 # Usage
 
+
+Download video and convert to mp3 (Requires ffmpeg)
+
+```bash
+ name=`echo myfilename` && ytdl http://www.youtube.com/watch?v=_HSylqgVYQI > $name.mp4 && ffmpeg -i $name.mp4 -b:a 192K -vn $name.mp3 && rm -rf $name.mp4
+
+```
```

Comment:
Would prefer this to be placed after the example below, since the example below is the most basic example of how someone would use ytdl

Reply:
Moved below the other simpler example.